great-idea/index.html

```diff
@@ -17,7 +17,87 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+           
+    <div class="navigation">  
+      <nav>
+        <a href="#">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+           <img class="logo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo.">
+   </div>
+   
+   <header>
+     <div class="header">
+    <div class="demand">
+     <h1> Innovation<br>
+     On<br>
+     Demand<br>
+     </h1>
+       </div>
       
     
+       
+     
+     <img class="top-image"src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/header-img.png" alt="Image of a code snippet.">
+     </div>
+    
+     
+    <div class="click">
+     <button> Get Started </button>
+    </div>
+     
+    </header>
+    
+   <section class="top-content">    
+     <div class="top">
+     <h3> Features </h3>
+     Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+     </div>
+   
+     <div class="top">
+     <h3> About </h3>
+     <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+     </div>  
+   </section>
+     
+     <img class="middle-img" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+       
+   <section class="bottom-content">    
+     <div class="bottom1">
+     <h3> Services </h3>
+     Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+     </div>
+   
+     <div class="bottom2">
+     <h3> Product </h3>
+     Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+     </div>
+   
+     <div class="bottom3">
+       <h3>Vision</h3>
+     Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+       <div>
+    </section> 
+     
+   
+     <h3 class="contact-title"> Contact </h3>
+    <div class="contact">
+     <p>123 Way 456 Street</p>
+     Somewhere, USA</p>
+     
+     <p>1 (888) 888-8888</p>
+     
+     <p>sales@greatidea.io</p>
```

Comment:
You may want to look into a more efficient way of doing this. Instead of having the seperate P tags. There again are lots of ways to skin this cat but the one I would have used is: 


I would move the contact h3 into the contact div. Instead of all the P tags from line 89 down to 94 

`<address>
          123 Way 456 Street<br>
          Somewhere, USA<br>
          <strong>1 (888) 888-8888</strong><br>
          <contact>sales@greatidea.io</contact>
        </address>`